lib/tasks/invites.rake

```diff
@@ -3,6 +3,6 @@
 namespace :invites do
   desc "Send invites"
   task :send_invites, [:emails] => :environment do |_task, args|
-    InviteParticipants.new.run(args.emails.split)
```

Comment:
What's the significance of removing the new?

Reply:
You can very easily stub and test class methods. Stubbing and testing instance methods did not seem nearly as easy when I tried it, so I decided to switch to using class methods instead.